plugins/UsersManager/Model.php

```diff
@@ -113,6 +113,39 @@ public function getUsersSitesFromAccess($access)
         return $return;
     }
 
+    /**
+     * @param string $userLogin
+     * @param int $idSite
+     *
+     * @return array|null The list of access for the given user login and id site
+     *
+     * @throws \Piwik\Tracker\Db\DbException
+     */
+    public function getAccessForUserForSite(string $userLogin, int $idSite): ?array
+    {
+        $db = $this->getDb();
+
+        $accessResults = $db->fetchAll(
+            "SELECT access FROM " . Common::prefixTable("access") . " WHERE login = ? AND idsite = ?",
+            [
+                $userLogin,
+                $idSite
+            ]
+        );
+
+        if ($accessResults === false) {
+            return null;
+        }
+
+        $accessList = [];
+
+        foreach ($accessResults as $accessResult) {
+            $accessList[] = $accessResult['access'];
+        }
+
+        return $accessList;
```

Comment:
Wouldn't that be the same as using
```suggestion
        return array_column($accessResults, 'access');
```